curriculum/challenges/english/25-front-end-development/lecture-introduction-to-python/67fe85a3db9bad35f2b6a2bd.md

```diff
@@ -1,17 +1,233 @@
 ---
 id: 67fe85a3db9bad35f2b6a2bd
 title: How Do Conditional Statements and Logical Operators Work?
-challengeType: 11
+challengeType: 19
 videoId: nVAaxZ34khk
 dashedName: how-do-conditional-statements-and-logical-operators-work
 ---
 
 # --description--
 
-Watch the video or read the transcript and answer the questions below.
+The video for this lecture isn't available yet, one will be available soon. Here is a transcript of the lecture for now:
 
 # --transcript--
 
+Conditional statements, or conditionals, let you control the flow of your program based on whether certain conditions are true or false.
```

Comment:
This script has not been approved yet and some parts of it will change. Writing this as a reminder to update it when it's time.